src/Compiler/Microsoft.AspNetCore.Razor.Language/src/CodeGeneration/CodeWriterExtensions.cs

```diff
@@ -398,10 +398,22 @@ private static CSharpCodeWritingScope BuildLambda(CodeWriter writer, bool async,
         return scope;
     }
 
-    public static CSharpCodeWritingScope BuildNamespace(this CodeWriter writer, string name)
+    public static CSharpCodeWritingScope BuildNamespace(this CodeWriter writer, string name, SourceSpan? span, CodeRenderingContext context)
     {
-        writer.Write("namespace ").WriteLine(name);
-
+        writer.Write("namespace ");
+        if (context.Options.DesignTime || span is null)
```

Comment:
Because for the foreseeable future we have to run both in fuse and non-fuse mode. That means we don't want to change the design time code at all. 

Eventually when fuse is done and we're happy we will burn every instance of `DesignTime` with fire.